libavfilter/dnn_filter_common.h

```diff
@@ -30,18 +30,20 @@ typedef struct DnnContext {
     char *model_filename;
     DNNBackendType backend_type;
     char *model_inputname;
-    char *model_outputname;
+    char *model_outputnames_expr;
```

Comment:
char *model_outputs ?

Reply:
changed its' name into 'model_outputnames_string', i see some code using the name like 'xxx_string', maybe it's acceptable
plus, i changed the corresponding func name to from 'separate_expression' to 'separate_output_names'